packages/blazor-workspace/BlazorWorkspace.sln

```diff
@@ -27,6 +27,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SprightBlazor.Tests", "Test
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SprightBlazor.Tests.Acceptance", "Tests\SprightBlazor.Tests.Acceptance\SprightBlazor.Tests.Acceptance.csproj", "{8E335572-CD8B-4879-8760-73416CF103B1}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NimbleBlazor.Tests.Acceptance.Shared", "Tests\NimbleBlazor.Tests.Acceptance.Shared\NimbleBlazor.Tests.Acceptance.Shared.csproj", "{D94A4535-51B3-4E70-9582-3F3A44C7A798}"
```

Comment:
I named the new project `NimbleBlazor.Tests.Acceptance.Shared`, which isn't great, since it isn't just for `NimbleBlazor` acceptance tests, but everything else I considered also had problems. I'm open to suggestions.

Reply:
Don't have a strong opinion since it's not a public assembly and I'm not familiar with conventions for C# namespaces or what the "problems" are in sharing (technical vs naming).
Some ideas:
- `BlazorWorkspace.Tests.Acceptance.Shared` (The goal of this assembly would be for sharing, seems redundant to add Shared in this case?)
- `BlazorWorkspace.Tests.Acceptance` (feels a little weird, like it should be tests, not test infra)
- `BlazorWorkspace.Testing.Acceptance` (like how we have Testing vs Tests in other packages)
- `Nimble.Testing.Acceptance`
- `NI.Testing.Acceptance`
- `Testing.Acceptance`

Reply:
Think I'm partial to `BlazorWorkspace.Testing.Acceptance` (tools of the workspace) or `NimbleBlazor.Testing.Acceptance` (tools shared by nimble blazor) but don't feel strongly

Reply:
`BlazorWorkspace.Testing.Acceptance` seems reasonable to me, so I'd vote for that too.

(Having `NimbleBlazor` or `.Tests.` in the name doesn't seem right)